cli/i18n/ignore/pl.php

```diff
@@ -1,6 +1,8 @@
 <?php
 
 return array(
+	'gen.date.format_date',
+	'gen.date.format_date_hour',
```

Comment:
We'll have to wait a bit for Travis, but I suspect/hope it'll complain this isn't alphabetical. ;-)